src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -144,82 +176,61 @@ private function dumpSitemap(PortalInformation $portalInformation)
      */
     private function dumpProviderSitemap($alias, PortalInformation $portalInformation)
     {
-        $provider = $this->sitemapProviderPool->getProvider($alias);
-        $maxPage = $provider->getMaxPage();
+        $page = 1;
+        do {
+            $sitemap = $this->sitemapRenderer->renderSitemap(
+                $alias,
+                $page,
+                $portalInformation->getLocale(),
+                $portalInformation->getPortal(),
+                $portalInformation->getHost(),
+                $this->scheme
```

Comment:
yes because currently we have no config for that. i would leave it like this for now.